src/locales/en.json

```diff
@@ -281,13 +289,24 @@
       "nonFrontline": "An average person in your area who works from home"
     },
     "plane": "An airplane",
+    "plane_breakdown": "for plane",
```

Comment:
For the airplane's air filtration system

Reply:
long